pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentDispatcherMultipleConsumers.java

```diff
@@ -93,7 +93,7 @@ public class PersistentDispatcherMultipleConsumers extends AbstractDispatcherMul
     protected volatile PositionImpl minReplayedPosition = null;
     protected boolean shouldRewindBeforeReadingOrReplaying = false;
     protected final String name;
-    protected volatile boolean sendInProgress;
+    protected boolean sendInProgress;
```

Comment:
I think if `protected boolean` works, `protected volatile boolean` doesn't cause regression?

Reply:
The usage of `volatile` did not cause a regression. It simply was unnecessary because `sendInProgress` is only ever updated within the `synchronized (this)` block.